opentaskspal/src/main/java/org/dmfs/android/contentpal/operations/AssertNoRow.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.android.contentpal.operations;
+
+import org.dmfs.android.contentpal.RowData;
+import org.dmfs.android.contentpal.Table;
+
+
+/**
+ * @author Gabor Keszthelyi
+ */
+// TODO This can probably use the Assert that receives RowSnapshot when that's available, instead of querying for RowData
+public final class AssertNoRow<T> extends DelegatingOperation<T>
+{
+    public AssertNoRow(Table<T> table, RowData<T> rowData)
+    {
+        super(new Counted<T>(0, new BulkAssert<T>(table, rowData)));
```

Comment:
Again, this is the same as `AssertEmptyTable`. It always fails if the given `Table` is not empty.

Reply:
This has been removed already.